docs/api_key.md

```diff
@@ -26,7 +26,7 @@ Setup a virtual environment, install the dependencies, and activate the virtual
     export AZURE_OPENAI_API_KEY="YOUR_API_KEY"
     export OPENAI_API_VERSION="2025-04-01-preview"
     export AZURE_OPENAI_ENDPOINT="https://YOUR_RESOURCE_NAME.openai.azure.com/"
-    export AZURE_DEPLOYMENT_NAME="gpt-4o"
+    export AZURE_OPENAI_DEPLOYMENT_NAME="gpt-4o"
```

Comment:
This makes it look like people should set a model name.
Is that really what we want? (I don't know). Maybe add a comment above.